website/docs/r/nrql_alert_condition.html.markdown

```diff
@@ -246,9 +246,6 @@ $ terraform import newrelic_nrql_alert_condition.foo 538291:6789035:outlier
 
 ~> **NOTE:** The value of `conditionType` in the import composite ID must be a valid condition type - `static`, `baseline`, or `outlier.` Also note that deprecated arguments will *not* be set when importing.
 
-The actual values for `policy_id` and `condition_id` can be retrieved from the following New Relic URL when viewing the NRQL alert condition you want to import:
```

Comment:
Can you provide an alternative on how customers could find the `policy_id` and `condition_id`?